_posts/2020-09-01-weather-prediction/weather_prediction_deep_learning.Rmd

```diff
@@ -0,0 +1,693 @@
+century, this is bound to be a rhetorical question. In the 1930s, when German poet Bertolt Brecht wrote the famous lines
+
+> Was sind das für Zeiten, wo\
+> Ein Gespräch über Bäume fast ein Verbrechen ist\
+> Weil es ein Schweigen über so viele Untaten einschließt![^1]
+
+[^1]: *An die Nachgeborenen, 1934-38.* The atrocities referred to are those of Nazi Germany.
+
+("What kind of times are these, where a conversation about trees is almost a crime, for it means silence about so many
+atrocities!"),
+
+he couldn't have anticipated the responses he would get in the second half of that century, with trees symbolizing, as well as
+literally falling victim to, environmental pollution and climate change.
+
+Today, no lengthy justification is needed as to why prediction of atmospheric states is vital: Due to global warming,
+frequency and intensity of severe weather conditions -- droughts, wildfires, hurricanes, heatwaves -- have risen and will
+continue to rise. And while accurate forecasts don't change those events per se, they constitute essential information in
+mitigating their consequences. This goes for atmospheric forecasts on all scales: from so-called "nowcasting" (operating on a
+range of about six hours) over medium-range (three to five days) and sub-seasonal (weekly/monthly) to climate forecasts
+(concerned with years and decades). Medium-range forecasts especially are extremely important in acute disaster prevention.
+
+This post will show how deep learning (DL) methods can be used to generate atmospheric forecasts, using a newly published
+[benchmark dataset](https://github.com/pangeo-data/WeatherBench)[@weatherbench]. Future posts may refine the model used here
+and/or discuss the role of DL ("AI") in mitigating climate change -- and its implications -- more globally. And coming back to
+Bertolt Brecht: While this text may be about trees, climate change should not remain the only societal topic addressed on this
+blog.
+
+That said, let's put the current endeavor in context. In a way, we have here the usual *dejà vu* of using DL as a
+black-box-like, magic instrument on a task where *human knowledge* used to be required. Of course, this characterization is
+overly dichotomizing; many choices are to be made in creating DL models, and performance is necessarily constrained by
+available algorithms -- which may, or may not, fit the domain to be modeled to a sufficient degree.
+
+If you've started learning about image recognition rather recently, you may well have been using DL methods from the outset,
+and not have heard much about the rich set of feature engineering methods developed in pre-DL image recognition. In the
+context of atmospheric prediction, then, let's begin by asking: How in the world did they do that *before*?
+
+## Numerical weather prediction in a nutshell
+
+It is not like machine learning and/or statistics are not already used in numerical weather prediction -- on the contrary. For
```

Comment:
Rather than have the double negative here, I'd just state that: _Machine learning and/or statistics are already used in numerical weather prediction._